aspnet/publishing/iis.rst

```diff
@@ -126,7 +132,7 @@ Troubleshooting:
 
 - If your application uses the `.UseUrls(...)` extension on `WebHostBuilder`, make sure you have positioned the `.UseUrls(...)` extension before the `.UseIISIntegration()` extension on `WebHostBuilder`. `.UseIISIntegration()` must overwrite any values you provide in `.UseUrls(...)` in order for the reverse-proxy to succeed.
 
-Issue #2
+Issue #3
```

Comment:
@Rick-Anderson @danroth27 I just got back to this. I'm having trouble. It's tough to title these ... There's too much info to title them easily when they don't have short behaviors and/or short possible cause lists.

Giving each a title based on browser response alone doesn't separate them. To give them titles based on all three behavioral classifications (browser response, ANCM log, and Application log) would make the titles too long. Giving them titles based on the possible causes ... also lists of possible causes in some cases ... also makes them too long.

This is why in the original doc I formatted them as three bulleted items to match the behavior followed by indented bullets for the possible causes. I thought that that was the simplest way to run down groups of three behaviors for the dev to match.

We could remove the "Issue #" from each and use a horizontal rule to separate them.